sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -870,21 +870,21 @@ def create_issuer(self, issuer_name, provider, **kwargs):
             issuer_credentials = self._client.models.IssuerCredentials(account_id=account_id, password=password)
         else:
             issuer_credentials = None
-        if admin_contacts and admin_contacts[0]:
-            admin_contacts_to_pass = [
+        if admin_contacts:
+            admin_details = [
                 self._client.models.AdministratorDetails(
-                    first_name=admin_detail.first_name,
-                    last_name=admin_detail.last_name,
-                    email_address=admin_detail.email,
-                    phone=admin_detail.phone,
+                    first_name=contact.first_name,
+                    last_name=contact.last_name,
+                    email_address=contact.email,
+                    phone=contact.phone,
                 )
-                for admin_detail in admin_contacts
+                for contact in admin_contacts
             ]
         else:
-            admin_contacts_to_pass = admin_contacts
+            admin_details = None
         if organization_id or admin_contacts:
```

Comment:
`if organization_id or admin_details`? Consider what happens when we get `admin_contacts=[]`.

Reply:
thanks so much for catching this!